src/main/kotlin/com/terraformation/backend/documentproducer/db/VariableStore.kt

```diff
@@ -77,6 +78,13 @@ class VariableStore(
    */
   private val replacements = ConcurrentHashMap<VariableId, List<VariableId>>()
 
+  fun fetchDeliverableVariableIds(deliverableId: DeliverableId): List<VariableId> =
+      dslContext
+          .select(VARIABLES.ID)
+          .from(VARIABLES)
+          .where(VARIABLES.DELIVERABLE_ID.eq(deliverableId))
+          .fetch(VARIABLES.ID.asNonNullable())
```

Comment:
Would `orderBy(VARIABLES.DELIVERABLE_POSITION)` make sense here so the IDs are guaranteed to be in the correct order?